### PR TITLE
docs: envs documentation

### PR DESCRIPTION
**Pull request by rixcian**
This PR should resolve the #47 issue.

I've noticed that the ENVs are quite well documented in the official docs. So I've just created a `.env.example` file with all possible ENVs and their description.

I've also created an `env.md` file where all the ENVs are specified in the markdown table. It's up to you if you want to use the table in some different place in the docs.

I've heard that @Bobinstein is working on docs, so his feedback will be more than appreciated.

### CODE REVIEW

File: docs/envs.md
Comment (Bobinstein) [inline docs/envs.md:1]: you can add 
LOG_FORMAT  |  string   | "simple"  | sets the format of output logs, accepts "simple" and "json" 
INSTANCE_ID   |  string   | ""             | Adds an "instanceId" field to output logs

File: .env.example
Comment (Bobinstein) [inline .env.example:1]: wrapping the values for ANS104_UNBUNDLE_FILTER and ANS104_INDEX_FILTER in single quotes can lead to parsing errors. The values do not need to be wrapped in any quotes.

File: docs/envs.md
Comment (rixcian) [reply docs/envs.md:1]: I've added the missing ENVs.

File: .env.example
Comment (rixcian) [reply .env.example:1]: I've removed the single quotes.

**Comment by djwhitt** [comment]
Thanks, @rixcian! Rebased and merged. Worth noting I made a couple of minor corrections in the process:

- https://github.com/ar-io/ar-io-node/commit/38210b9979d50ceb00a24feb12bd244b09f1d3c4
- https://github.com/ar-io/ar-io-node/commit/eb843a76cf7389fefac5df908021edc1a1a754b7

**Pull request closed without merging**